src/main/java/com/google/sps/model/Catagory.java

```diff
@@ -0,0 +1,5 @@
+package com.google.sps.model;
+
+public enum Catagory {
```

Comment:
If you plan to have multiple enums like this, you can also put them in  class like this, so that you don't have to create a separate class for each one

public class Types {
  public enum Category {

  }

  public enum SomeOtherType {

  }
}

Reply:
Oh that makes a lot of sense. Is this preferred over a module with enums classes?